tests/org.eclipse.n4js.xpect.ui.tests/testdata/project_circular_dependencies/packageA.json.xt

```diff
@@ -31,7 +31,7 @@
 // Testing cycle-detection IDEBUG-266
 
 {
-	// XPECT errors --> "The project configuration contains a dependency cycle of projects [A] -> C -> B -> [A]." at ""A""
+	// TODO: GH-1500 XPECT FIXME errors --> "The project configuration contains a dependency cycle of projects [A] -> C -> B -> [A]." at ""A""
```

Comment:
Since you won't fix this as part of `GH-1500`, please create a sub task for the cycle detection improvements and link its task ID here (or maybe use the next general LSP task for that).
(also applies to 3 other files in same folder)